Formula/x/xnvme.rb

```diff
@@ -0,0 +1,26 @@
+class Xnvme < Formula
+  desc "Description of your project"
+  homepage "https://xnvme.io/"
+  url "https://github.com/OpenMPDK/xNVMe/releases/download/v0.7.2/xnvme-0.7.2.tar.gz"
+  sha256 "1cb849b537cfddc15d82b8f4622fe3f999b4c7c0542c55b8d09b485e016e942e"
+  license "BSD-3-Clause"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "python@3.11" => :build
```

Comment:
Can it work with python 3.12?

Reply:
It should. Python3 is needed since the meson-build calls out to the Python interpreter. I had to add it as a dependency otherwise there was build-failures on the ci. I tried just adding "python@3" but that gave style-check issue so ended up using the specific version. Is it possible to specify python>=3.9?

Reply:
I tried right now to just put in ``python`` to see what the CI say about that...

Reply:
It is telling me this: ``* Dependency 'python' is an alias; use the canonical name 'python@3.11'.``
But adding python@3.11, is not the way to go?

Reply:
> Is it possible to specify python>=3.9?

Nope, with Homebrew, you have to specify _which_ Python version you want to depend on. I think @SMillerDev is suggesting this change:

```suggestion
  depends_on "python@3.12" => :build
```

Reply:
Aight, bumped it to 3.12.